src/util.js

```diff
@@ -869,3 +876,23 @@ export function focusTextArea(element) {
 
     return textAreaEl;
 }
+
+/**
+ * Generates a blank mobile annotation dialog
+ *
+ * @return {HTMLElement} Blank mobile annotation dialog
+ */
+export function generateMobileDialogEl() {
+    const el = document.createElement('div');
+    el.classList.remove(CLASS_ANNOTATION_PLAIN_HIGHLIGHT);
```

Comment:
Remove these 2 lines. You shouldn't need to remove classes on a fresh div

Reply:
👍This was related to previous code. Removing it now